lib/internal/errors.js

```diff
@@ -28,7 +28,6 @@ function makeNodeError(Base) {
     constructor(key, ...args) {
       super(message(key, args));
       this[kCode] = key;
-      Error.captureStackTrace(this, NodeError);
```

Comment:
I'm -1 on removing this line. The fact that `NodeError` is being used in here should not be visible to the end user. This intentionally removes that frame from the stack trace to provide that transparency.